sdk/containerregistry/azure-containerregistry/azure/containerregistry/_generated/aio/operations/_authentication_operations.py

```diff
@@ -178,3 +195,59 @@ async def exchange_acr_refresh_token_for_acr_access_token(
 
     exchange_acr_refresh_token_for_acr_access_token.metadata = {'url': '/oauth2/token'}  # type: ignore
 
+
+    @distributed_trace_async
+    async def get_acr_access_token_from_login(
```

Comment:
This isn't exposed in the sdk we show users, should it be?

Reply:
We are not using username/pwd to authenticate as service team recommended. I didn't do any config so it still generated. AFAIK, in js it removed this operation: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/containerregistry/container-registry/swagger#remove-authentication_getacraccesstokenfromlogin-operation
Do u mind to leave it there or better to get it removed?

Reply:
I dont think it matters too much personally, it would be nicer to not have it since it reduces file size by a little bit, but definitely not highest priority

Reply:
Make sense, will update in this PR.